mitylib/util.py

```diff
@@ -148,21 +160,23 @@ def write_merged_vcf(new_vcf, out_file, genome_file='mitylib/reference/b37d5.gen
             myfile.write(vcf_line + '\n')
     gsort_vcf(f, out_file, genome_file=genome_file)
 
-def create_genome_file(vcf_file, genome_file):
-    """
-    gsort (https://github.com/brentp/gsort) requires a '.genome'
-    file to tell it how to sort the vcf records. This function creates a
-    '.genome' file in the same order as the contig lines in the vcf header.
-
-    :param vcf_file: a vcf file with the correct contig names
-    :param genome_file: the resulting .genome file
-    :return: None. this creates a '.genome' file
-    """
-
-    vcf = pyvcf.Reader(filename=vcf_file)
-    with open(genome_file, mode='wt', encoding='utf-8') as genome_file:
-        for contig in vcf.contigs.keys():
-            genome_file.write('\t'.join(vcf.contigs[contig]))
+# REMOVE: This function isn't called anywhere???
+# def create_genome_file(vcf_file, genome_file):
```

Comment:
Just remove this if it is not being used. If required we can get it from a previous commit